health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/utils/setupCampaignHelpers.js

```
@@ -300,12 +300,15 @@ export const updateUrlParams = (params) => {
   export const transformDraftDataToFormData =(draftData,projectType)=>{
 
     const delivery = Array.isArray(draftData?.deliveryRules) ? draftData?.deliveryRules : [];
-    const filteredProjectType = projectType?.["HCM-PROJECT-TYPES"]?.projectTypes?.filter((i) => i?.code === draftData?.projectType);
+    const filteredProjectType = projectType?.["HCM-PROJECT-TYPES"]?.projectTypes?.find((i) => i?.code === draftData?.projectType);
     const restructureFormData = {
       HCM_CAMPAIGN_TYPE: { projectType: filteredProjectType?.[0] },
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)